app/addClothe.tsx

```diff
@@ -1,59 +1,242 @@
+    const [selectedColor, setSelectedColor] = useState(""); // Default value
+    const [selectedSize, setSelectedSize] = useState(-1); // Default value
+    
+    
+    const handleCancel = () => {
+        navigation.goBack();
+    };
+    
     async function addClothe() {
+        // Update the properties of the clothe object directly with the selected values
+        clothe.name = selectedName;
+        clothe.description = selectedDescription;
+        clothe.link = selectedLink;
+        clothe.type = selectedType;
+        clothe.color = selectedColor;
+        clothe.sleeveSize = selectedSize;
+        
+        
+        // Add the updated clothe object to the database
         const success = await DatabaseController.addClothe(clothe);
-
+        
+        console.log("SELECTED TYPE: " + selectedType)
         if (success) {
-            console.log("Successfully added clothe!")
+            console.log("Successfully added clothe of name " + clothe.name + " of type " + clothe.type + " of color " + clothe.color + " of sleeve size " + clothe.sleeveSize + " with link " + clothe.link)
         } else {
             console.log("Failed to add clothe!")
         }
-
+    
         navigation.goBack();
     }
 
     return (
         <View style={styles.container}>
-            <TextInput style={styles.input} placeholder="Clothe name"
-                       onChangeText={(text) => clothe.name = text}/>
-            <View style={styles.separator} lightColor="#eee" darkColor="rgba(255,255,255,0.1)"/>
+            <View style={styles.photoContainer}>
+                <Image style={styles.image} source={{ uri: selectedLink }} /> 
+                {/*The URL here should be the URI from the photo capture feature. Simply make the selectedLink variable be the URI and that should make the class get that data*/}
+                <View style={styles.greyBottom}/>
+                <TextInput style={styles.input} placeholder="Type Item Name Here"
+                    onChangeText={(text) => setSelectedName(text)} // Update the selectedName state
+                    />       
+            </View>
+
+            <View style={styles.filterSquare}>
+                <Text style={styles.filterText}>Tags</Text>
+                <View style={styles.pickerBox}>
 
-            <TextInput style={styles.input} placeholder="Clothe description"
-                       onChangeText={(text) => clothe.description = text}/>
-            <View style={styles.separator} lightColor="#eee" darkColor="rgba(255,255,255,0.1)"/>
+                    <Picker style={styles.picker} itemStyle={styles.pick}
+                        selectedValue={selectedType} 
+                        onValueChange={(itemValue: Clothe.Type) => 
+                            setSelectedType(itemValue) // Update the selectedType state 
+                    }>
+                        <Picker.Item label="Type" value={Clothe.Type.Unknown} />
```

Comment:
I think it'd be better for these labels to be `Unknown`.

Reply:
I refer to the comment with the picture:

The reason as to why the label is set to "Type", is because it says what the picker is for. It's like it's default position. I would have to style further the Android and iOS to fix this, and thought it would be better to just leave the placeholders say what the picker is about. And leave them at value "unknown". I could fix this, but it would take too much time on something so small in my opinion.

Let me know what you think.
![IMG_2859](https://github.com/uprm-inso4115-2023-2024-s2/semester-project-photofit-a-closet-inventory-app/assets/93483234/828c551f-22ad-4e17-aea0-36c2c4d05383)

Reply:
Followed your advice and changed to unknown.